dbms/src/Functions/tests/gtest_strings_elt.cpp

```diff
@@ -0,0 +1,195 @@
+               createColumn<Nullable<Int8>>({1, 2, 1, 100, 2, 1, 0, -2}),
+               createColumn<Nullable<String>>({"abc", "123", "", "", {}, "", "a", "c"}),
+               createColumn<Nullable<String>>({"def", "321", {}, "", "", "", "b", "d"}));
+
+    ASSERT_ELT(createColumn<Nullable<String>>({"abc", "321", "", {}, "", "", {}, "c"}),
+               createColumn<Nullable<UInt8>>({1, 2, 1, 100, 2, 1, 0, 1}),
+               createColumn<Nullable<String>>({"abc", "123", "", "", {}, "", "a", "c"}),
+               createColumn<Nullable<String>>({"def", "321", {}, "", "", "", "b", "d"}));
+
+    ASSERT_ELT(createColumn<Nullable<String>>({"abc", "321", "", {}, "", "", "a", {}}),
+               createColumn<Nullable<Int16>>({1, 2, 3, 100, 3, 2, 1, -1}),
+               createColumn<Nullable<String>>({"abc", "123", "", "", {}, "", "a", "c"}),
+               createColumn<Nullable<String>>({"def", "321", {}, "", "", "", "b", "d"}),
+               createColumn<Nullable<String>>({"ghi", "456", "", "", "", "", "e", "f"}));
+
+    ASSERT_ELT(createColumn<Nullable<String>>({"abc", "321", "", {}, "", "", {}, "c"}),
+               createColumn<Nullable<UInt16>>({1, 2, 1, 100, 2, 1, 0, 1}),
+               createColumn<Nullable<String>>({"abc", "123", "", "", {}, "", "a", "c"}),
+               createColumn<Nullable<String>>({"def", "321", {}, "", "", "", "b", "d"}));
+
+    ASSERT_ELT(createColumn<Nullable<String>>({"abc", "321", "", {}, "", "", "a", {}}),
+               createColumn<Nullable<Int32>>({1, 2, 3, 100, 3, 2, 1, -1}),
+               createColumn<Nullable<String>>({"abc", "123", "", "", {}, "", "a", "c"}),
+               createColumn<Nullable<String>>({"def", "321", {}, "", "", "", "b", "d"}),
+               createColumn<Nullable<String>>({"ghi", "456", "", "", "", "", "e", "f"}));
+
+    ASSERT_ELT(createColumn<Nullable<String>>({"abc", "321", "", {}, "", "", {}, "c"}),
+               createColumn<Nullable<UInt32>>({1, 2, 1, 100, 2, 1, 0, 1}),
+               createColumn<Nullable<String>>({"abc", "123", "", "", {}, "", "a", "c"}),
+               createColumn<Nullable<String>>({"def", "321", {}, "", "", "", "b", "d"}));
+
+    ASSERT_ELT(createColumn<Nullable<String>>({"abc", "321", "", {}, "", "", "a", {}}),
+               createColumn<Nullable<Int64>>({1, 2, 3, 100, 3, 2, 1, -1}),
+               createColumn<Nullable<String>>({"abc", "123", "", "", {}, "", "a", "c"}),
+               createColumn<Nullable<String>>({"def", "321", {}, "", "", "", "b", "d"}),
+               createColumn<Nullable<String>>({"ghi", "456", "", "", "", "", "e", "f"}));
+
+    ASSERT_ELT(createColumn<Nullable<String>>({"abc", "321", "", {}, "", "", {}, "c"}),
+               createColumn<Nullable<UInt64>>({1, 2, 1, 100, 2, 1, 0, 1}),
+               createColumn<Nullable<String>>({"abc", "123", "", "", {}, "", "a", "c"}),
+               createColumn<Nullable<String>>({"def", "321", {}, "", "", "", "b", "d"}));
```

Comment:
and add tests for `createConstColumn<Nullable<String>>`?

Reply:
Sure